### PR TITLE
feat: add custom expand style for typography.paragraph

### LINKED ISSUE #17975: 在 Typography.Paragraph 自动溢出省略中，能修改展开按钮的样式吗？我想做成像Dropdown 这样的下拉效果, 或者Dropdown 能限制隐藏行数吗
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
能让我们做出更加个性化的网站

### What does the proposed API look like?
希望能提供一个属性来修改效果

### PR DESCRIPTION
**Pull request by fireairforce**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close #17975

### 💡 Background and solution

add custom expand style for typography.paragraph

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/typography/index.en-US.md](https://github.com/fireairforce/ant-design/blob/feature-typography/components/typography/index.en-US.md)
[View rendered components/typography/index.zh-CN.md](https://github.com/fireairforce/ant-design/blob/feature-typography/components/typography/index.zh-CN.md)

### CODE REVIEW

**Comment by fireairforce** [comment]
Shoud I add some test cases or example for this change?

**Comment by afc163** [comment]
ci broken

**Comment by fireairforce** [comment]
fixed~

**Comment by afc163** [comment]
测试覆盖率要补一下。

**Comment by fireairforce** [comment]
好的。

**Comment by fireairforce** [comment]
@afc163 PTAL

File: components/typography/index.zh-CN.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ cols: 1
 | delete | 添加删除线样式 | boolean | false |  |
 | disabled | 禁用文本 | boolean | false |  |
 | editable | 是否可编辑，为对象时可对编辑进行控制 | boolean \| { editing: boolean, onStart: Function, onChange: Function(string) } | false |  |
-| ellipsis | 自动溢出省略，为对象时可设置省略行数、是否可展开、添加后缀等 | boolean \| { rows: number, expandable: boolean, suffix: string, onExpand: Function(event), onEllipsis: Function(ellipsis) } | false | onEllipsis: 4.2.0 |
+| ellipsis | 自动溢出省略，为对象时可设置省略行数、是否可展开、添加后缀等 | boolean \| { rows: number, expandable: boolean, suffix: string, symbol: string | React.ReactNode, onExpand: Function(event), onEllipsis: Function(ellipsis) } | false | onEllipsis: 4.2.0 |
```
Comment (afc163) [inline components/typography/index.zh-CN.md:57]:
```suggestion
| ellipsis | 自动溢出省略，为对象时可设置省略行数、是否可展开、添加后缀等 | boolean \| { rows: number, expandable: boolean, suffix: string, symbol: React.ReactNode, onExpand: Function(event), onEllipsis: Function(ellipsis) } | false | onEllipsis: 4.2.0 |
```

**Comment by fireairforce** [comment]
done~

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
PR 主楼要补充一下 changelog 的表格，方便发布的同学。

**Comment by fireairforce** [comment]
好的，以后会注意的~

**Pull request merged**